### PR TITLE
Enock Khondowe 202025800044 & Samantha Rusike 202025800046: Mark Subm…

### PR DESCRIPTION
**Pull request by enockkays**
Bug No.177 Mark submission does not work (http://118.25.96.118/bugzilla/show_bug.cgi?id=177)
This bug was filed on 2020/09/26 by Lan Hui

> Mark Submission no longer working
> (Alas, it worked before.)
> 
> After I clicked the "Mark Submission" button, I saw this prompt:
> 
>     TypeError: $(...).dialog is not a function

The above-mentioned error has been resolved successfully in the Submissions.php file line 496 – 500.

'Submit Marking': function () {
                $('#submit-form').submit();
                //$('input[type=submit].default').submit();
                $(this).dialog('close');
            },

### CODE REVIEW

File: Submissions.php [hunk 1]
```diff
@@ -528,7 +528,7 @@ function updatev(id)
                                                                                                                                                                                                              title:'Update Report Visibility',
                                                                                                                                                                                                              buttons: {
             'Update': function () {
-                $('#frm').submit();
```
Comment (spm2020spring) [inline Submissions.php]: @enockkays  

Why has not the form id "frm" at line 523 not changed?

-Hui

**Comment by enockkays** [comment]
My colleague Samantha is working on it now she will make that change on git hub ASAP. Enock Khondowe 202025800044 -------- Original message --------From: Software Project Management <notifications@github.com>Date: Tue, 22 Dec 2020, 18:13To: lanlab-org/LRR <LRR@noreply.github.com>Cc: enockkays <enockkhondowe94@yahoo.com>, Mention <mention@noreply.github.com>Subject: Re: [lanlab-org/LRR] Enock Khondowe 202025800044 & Samantha Rusike 202025800046: Mark Subm… (#51)
@spm2020spring commented on this pull request.



In Submissions.php:
> @@ -528,7 +528,7 @@ function updatev(id)
                                                                                                                                                                                                              title:'Update Report Visibility',
                                                                                                                                                                                                              buttons: {
             'Update': function () {
-                $('#frm').submit();

@enockkays
Why has not the form id "frm" at line 523 not changed?
-Hui

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by enockkays** [comment]
老师好！The changes have been made to the branch name lrr-enockkays/Samantha. Looking forward to your favourable response thanks. Kind Regards, Enock Khondowe,202025800044. -------- Original message --------From: Software Project Management <notifications@github.com>Date: Tue, 22 Dec 2020, 18:13To: lanlab-org/LRR <LRR@noreply.github.com>Cc: enockkays <enockkhondowe94@yahoo.com>, Mention <mention@noreply.github.com>Subject: Re: [lanlab-org/LRR] Enock Khondowe 202025800044 & Samantha Rusike 202025800046: Mark Subm… (#51)
@spm2020spring commented on this pull request.



In Submissions.php:
> @@ -528,7 +528,7 @@ function updatev(id)
                                                                                                                                                                                                              title:'Update Report Visibility',
                                                                                                                                                                                                              buttons: {
             'Update': function () {
-                $('#frm').submit();

@enockkays
Why has not the form id "frm" at line 523 not changed?
-Hui

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by spm2020spring** [comment]
@enockkays  Why does your branch have conflicts now?

-Hui

File: Submissions.php [hunk 2]
```diff
@@ -528,7 +528,10 @@ function updatev(id)
                                                                                                                                                                                                              title:'Update Report Visibility',
                                                                                                                                                                                                              buttons: {
             'Update': function () {
+
                 $('#submit-form').submit()
+                $('#submit-form').submit();
```
Comment (spm2020spring) [inline Submissions.php:531]: @enockkays  What is the point of having duplicate lines at line 532 and 533?  The line "$('#submit-form').submit()" appears twice at the two places.

-Hui

**Comment by enockkays** [comment]
老师好！
Thank you for the observation, the duplicate line was done by mistake and it has therefore been removed.

Kind Regards,
Enock Khondowe.
202025800044.

Sent from Mail for Windows 10

**Comment by spm2020spring** [comment]
@enockkays 

Curiously, after updating the visibility of a student's assignment (for reading 4) to Public, I got the following outcome.  That is, the count for each field becomes 0.


> Lab Report Visibility Updated
> Lab Report Submissions (0)
> Un-Marked Submissions (0)
> Marked Submissions (0)
> Re-Marking Requests (0)
> View Course Groups
> 


-Hui

**Pull request merged**